gke/gpu/maxtext_chart/templates/maxtext.yaml

```diff
@@ -0,0 +1,144 @@
+{{- $requiredVar := .Values.cluster.nNodes | required ".Values.cluster.nNodes is required" -}}
```

Comment:
all fine for now but needs to go into XPK right? can you eventually own cleanup here?

Reply:
Ack.